infra/app/env-template/variables.tf

```diff
@@ -1,24 +1,42 @@
+##############################################
+## General variables
+##############################################
 variable "environment_name" {
   type        = string
   description = "name of the application environment"
 }
 
+
+##############################################
+## Variables for the participant app
+##############################################
 variable "participant_image_tag" {
   type        = string
   description = "Image tag to deploy to the environment for the participant service"
   default     = "latest"
 }
 
-variable "staff_image_tag" {
+variable "participant_url" {
   type        = string
-  description = "Image tag to deploy to the environment for the staff service"
-  default     = "latest"
+  description = "URL to access the application"
 }
 
-variable "analytics_image_tag" {
+variable "participant_max_upload_size_bytes" {
   type        = string
-  description = "Image tag to deploy to the environment for the analytics service"
-  default     = "latest"
+  description = "The maximum allowed size of a single file upload"
+  default     = "26_214_400"
```

Comment:
I actually don't know - does Javascript parseInt("26_214_400") as `26214400` ? (I know that `26_214_400` works)

Reply:
🤔 Good question. I thought it did, but now I'm second guessing myself. I'll double check.

Reply:
You were correct. Javascript `parseInt("26_214_400")` turns into `26`. I've changed this to just `"26214400"` which `parseInt()` does handle correctly. Good catch. Thank you!